dotcom-rendering/.storybook/mocks/jsdom.js

```diff
@@ -11,8 +11,18 @@
 export class JSDOM {
 	constructor() {}
 	window = window;
-}
 
-// Attaches static 'fragment' method for calling 'JSDOM.fragment(...)' (curently untested)
-const fragment = (...params) => new window.DocumentFragment(...params);
-Object.assign(JSDOM, fragment);
+	static fragment(html) {
+		try {
+			const domParser = new DOMParser();
+			const docNodes = domParser.parseFromString(html, 'text/html').body
```

Comment:
If you wanted to avoid the cost of creating a new `DOMParser`, you would have to take it out of this method.

```suggestion
			const docNodes = new DOMParser().parseFromString(html, 'text/html').body
```

Reply:
The benefit of extracting the `new DOMParser()` seem to be negligible at ~2% performance improvement:

<img width="1016" alt="image" src="https://github.com/guardian/dotcom-rendering/assets/76776/44aa1998-8024-40ba-a99e-6682ef545cb8">